src/main/groovy/org/mockito/release/gradle/ReleaseNotesFetcherTask.java

```diff
@@ -22,12 +22,17 @@
 public class ReleaseNotesFetcherTask extends DefaultTask {
 
     @Input private String previousVersion;
+    @Input private String version = getProject().getVersion().toString();
     @Input private String gitHubReadOnlyAuthToken;
     @Input private String gitHubRepository;
+    @Input private String tagPrefix = "v";
+    @Input private boolean onlyPullRequests = false;
+    @Input private File gitWorkDir = getProject().getRootDir();
+    @Input private Collection<String> gitHubLabels = Collections.emptyList();
     @OutputFile private File outputFile;
 
     /**
-     * GitHub read only authentication token needed for loading issue information from GitHub.
+     * See {@link ReleaseConfiguration.GitHub#getReadOnlyAuthToken()}
```

Comment:
Good idea to NOT copy paste javadoc, but reference to the relevant and detailed description in `ReleaseConfiguration`'s javadoc. 👍